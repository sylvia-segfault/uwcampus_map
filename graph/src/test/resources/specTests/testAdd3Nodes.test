# Create an empty graph
CreateGraph graph3

# Add 3 nodes to graph3
AddNode graph3 node1
AddNode graph3 node2
AddNode graph3 node3