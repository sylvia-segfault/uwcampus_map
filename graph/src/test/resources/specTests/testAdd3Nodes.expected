# Create an empty graph
created graph graph3

# Add 3 nodes to graph3
added node node1 to graph3
added node node2 to graph3
added node node3 to graph3